roles/sap_ha_pacemaker_cluster/vars/redhat.yml

```diff
@@ -19,3 +19,55 @@ __sap_ha_pacemaker_cluster_command:
   resource_start: "pcs resource enable"
   resource_defaults_show: "pcs resource defaults config"
   resource_defaults_update: "pcs resource defaults update"
+
+# Make sure that there is always the minimal default fed into the included role.
+# This is combined with the custom list 'sap_ha_pacemaker_cluster_fence_agent_packages'.
+sap_ha_pacemaker_cluster_fence_agent_minimal_packages:
+  - fence-agents-all
+
+# Dictionary with fence packages for each platform
+sap_ha_pacemaker_cluster_fence_agent_packages_dict:
+  cloud_aws:
+    - fence-agents-aws
+  cloud_gcp:
+    - fence-agents-gce
+  cloud_ibmcloud_powervs:
+    - fence-agents-ibm-powervs
+  cloud_ibmcloud_vs:
+    - fence-agents-ibm-vpc
+  cloud_msazure_vm:
+    - fence-agents-azure-arm
+  hyp_ibmpower_vm:
+    - fence-agents-lpar
+
+# Dictionary with extra platform specific packages
+__sap_ha_pacemaker_cluster_platform_extra_packages_dict:
+  cloud_aws:
+    - awscli
+  cloud_gcp:
+    - resource-agents-gcp
+  cloud_ibmcloud_powervs: []
+  cloud_ibmcloud_vs: []
+  cloud_msazure_vm:
+    - socat
+  hyp_ibmpower_vm: []
+
+# Dictionary with additional cluster packages for specific scenarios
+__sap_ha_pacemaker_cluster_sap_extra_packages_dict:
+  scaleout:
+    - resource-agents-sap-hana-scaleout
+  scaleup:
+    - resource-agents-sap-hana
+  nwas:
+    - resource-agents-sap
+
```

Comment:
@sean-freeman I will be updating support coverage in Readmes across whole project to reflect real supported versions (next PR to avoid getting delayed more). Plan is to support only SLES4SAP for HA and majority, SLES15 only for simple onehost scenarios.

This means there will not be extra repos added on suse as they come from registered product by default.

Reply:
I was mostly concerned with ensuring consistency of different Package Repos in case of variance in Cloud platforms (like that for AWS + RHEL), with a consistent code approach, and the code only there as a safety net in case someone has disabled a default / mandatory Package Repo :)

Defer for later